src/mSupplyMobileApp.js

```diff
@@ -83,21 +83,27 @@ class MSupplyMobileAppContainer extends React.Component {
     this.handleBackEvent = this.handleBackEvent.bind(this);
     this.getCanNavigateBack = this.getCanNavigateBack.bind(this);
     this.renderPageTitle = this.renderPageTitle.bind(this);
-    this.scheduler.schedule(this.synchronise,
-                            SYNC_INTERVAL);
+    this.scheduler.schedule(this.synchronise, SYNC_INTERVAL);
     this.scheduler.schedule(() => {
-      if (this.state.currentUser !== null) { // Only reauthenticate if currently logged in
+      if (this.state.currentUser !== null) {
+        // Only reauthenticate if currently logged in
         this.userAuthenticator.reauthenticate(this.onAuthentication);
       }
     }, AUTHENTICATION_INTERVAL);
   }
 
   componentDidMount() {
     BackHandler.addEventListener('hardwareBackPress', this.handleBackEvent);
+    if (this.state.isInitialised && this.synchroniser.isSyncing()) {
```

Comment:
App shouldn't close with back button. Should still be 'idle' in the background and should call mount when it's opened back up. I will make sure our testers today will try the following:
Press back button, go back in app, try to sync a record with -1 sequence number and see if updates.